src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java

```diff
@@ -252,15 +252,13 @@ public String createNewProject(final Model model, @ModelAttribute Project projec
 	 * @param model     model for view variables
 	 * @return name of the analysis view page
 	 */
-	@RequestMapping("/projects/{projectId}/analyses")
+	@RequestMapping("/projects/{projectId}/analyses/**")
 	public String getProjectAnalysisList(@PathVariable Long projectId, Principal principal, Model model) {
 		Project project = projectService.read(projectId);
 		projectControllerUtils.getProjectTemplateDetails(model, principal, project);
 		model.addAttribute("project", project);
```

Comment:
Let's get this off of here if we can.  You can wrap it in the redux-toolkit to get info about the project if the page needs it.  I have also been wrapping project entries with the Router to get the projectId if I need to to avoid any funky regex to get it.  This will help later on when we want to drop down to only one entry for a project.

Reply:
This is just being used on the Thymeleaf template (project_analyses.html) to get the project label. Should I just set the project label in the model instead of the whole project object?